src/rest/executor/http_utils.erl

```diff
@@ -0,0 +1,41 @@
+%%%-------------------------------------------------------------------
```

Comment:
Mam wrażenie, że już oglądałem ten plik? Dlaczego są takie dwa?

Reply:
bo on to przeniósł do aplikacji serwera (ściezki plików ;) )

Reply:
Exactly. gmm_client to CAŁKOWICIE OSOBNY PROJEKT, po prostu jest ulokowany w katalogu projektu gmm_server.

Btw. wiąże się z tym fakt, że dialyzer i wszystkie inne checki które zrobiłeś do "głównej" apki nie stosują się do klienta. Z tego też powodu był tam taki burdel jaki był. Polecam dorobić tam dialyzera, ale to nie moja działka